x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/details_step/additional_section.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useMemo, useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiAccordion, EuiSpacer } from '@elastic/eui';
+import type { MlUrlConfig } from '@kbn/ml-anomaly-utils';
+import type { DataFrameAnalyticsConfig } from '@kbn/ml-data-frame-analytics-utils';
+import { DeepPartial } from '../../../../../../../common/types/common';
+import { Description } from './description';
+import { CustomUrlsWrapper } from '../../../../../components/custom_urls';
+import {
+  getJobConfigFromFormState,
+  type State,
+} from '../../../analytics_management/hooks/use_create_analytics_form/state';
+import { ActionDispatchers } from '../../../analytics_management/hooks/use_create_analytics_form/actions';
+
+const buttonContent = i18n.translate(
+  'xpack.ml.dataframe.analytics.create.detailsStep.additionalSectionButton',
+  {
+    defaultMessage: 'Additional settings',
+  }
+);
+
+interface Props {
+  formState: State['form'];
+  setFormState: ActionDispatchers['setFormState'];
+}
+
+export const AdditionalSection: FC<Props> = ({ formState, setFormState }) => {
+  const [additionalExpanded, setAdditionalExpanded] = useState<boolean>(false);
+  const { _meta: formMeta } = formState;
+
+  const analyticsJob = useMemo(
+    () => getJobConfigFromFormState(formState) as DeepPartial<DataFrameAnalyticsConfig>,
+    [formState]
+  );
+  const setCustomUrls = (urls: MlUrlConfig[]) => {
+    setFormState({ _meta: { ...formMeta, custom_urls: urls } });
+  };
+
+  return (
+    <>
+      <EuiSpacer />
+      <EuiAccordion
+        id="additional-section"
```

Comment:
Do we need this `id`, it seems a bit unspecific, is it used for anything?

Reply:
It's required by the EUI component. I'm not sure how it's used internally.

Reply:
Looks like it gets applied to the content part and is used to references elements via aria tags. Since it's present in the DOM we should make it more unique, for example at least with an `ml` prefix.